config.py

```diff
@@ -384,6 +384,8 @@ def setup_extension_modules(self):
     if self.debug_mode:
       for k in self.compiler_flags:
         self.compiler_flags[k].append('-g')
+        ind = [i for i, item in enumerate(self.compiler_flags[k]) if item.startswith('-O')]
```

Comment:
I do believe there would be a more elegant way here for an advanced Python user. The object is to set the optimization -O0 when the debug-mode flag is on, rather than the default -O3.

Reply:
This is fine in my opinion. It's clear what the code is doing
However this needs to be two lines